### PR TITLE
Depression

### PR DESCRIPTION
**Pull request by bottlerocket**
Depression pages

### CODE REVIEW

File: assets/scss/modules/controls/_condition-navigation.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+nav.condition-navigation {
```
Comment (Demotive) [inline assets/scss/modules/controls/_condition-navigation.scss]: Do you need the element selector at the start? Can't it just be `.condition-navigation` ?

File: assets/scss/modules/typography/_lists.scss [hunk 2]
```diff
@@ -1,17 +1,17 @@
 ul, ol {
   margin: $leading 0 $leading*2 0;
-  padding-left: 1em;
-  list-style-type: square;
+  padding-left: 0px;
```
Comment (Demotive) [inline assets/scss/modules/typography/_lists.scss]: This is what's wiped out the bullets - leave it at `padding-left: 1em;` ?

As an aside, `0` is fine, you don't need to write `0px`

File: assets/scss/modules/typography/_lists.scss [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+  list-style-type: none;
```
Comment (Demotive) [inline assets/scss/modules/typography/_lists.scss]: Oh, this as well. This is zeroing out all lists? I'd suggest setting it to `disc` but that's a personal opinion.

File: assets/scss/modules/typography/_lists.scss [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
   letter-spacing: 0.01em;
   list-style-position: inside;
```
Comment (Demotive) [inline assets/scss/modules/typography/_lists.scss:6]: Should be `outside` I think. If a list item ever wraps over more than one line, the bullet will be visually lost...

File: assets/scss/modules/controls/_condition-navigation.scss (hunk 1 above)
Comment (bottlerocket) [inline assets/scss/modules/controls/_condition-navigation.scss]: correct. remove it.

File: assets/scss/modules/typography/_lists.scss (hunk 3 above)
Comment (bottlerocket) [inline assets/scss/modules/typography/_lists.scss]: yes, swapped it do disc for you....

File: assets/scss/modules/typography/_lists.scss (hunk 4 above)
Comment (bottlerocket) [inline assets/scss/modules/typography/_lists.scss:6]: not sure why I did that originally. I've changed it back to outside. Likewise, the list-style-type was set to none....no idea why that was in there. Might have been something I was thinking about for lists within the callout modules

File: assets/scss/modules/typography/_lists.scss (hunk 3 above)
Comment (Demotive) [inline assets/scss/modules/typography/_lists.scss]: 👍

**Pull request merged**